src/wirecloud/defaulttheme/static/css/header.scss

```diff
@@ -33,27 +33,38 @@
 #wirecloud_header .menu_wrapper > .menu {
     display: inline-block;
     background: #E4E6E2;
-    box-shadow: 0 3px 3px rgba(0, 0, 0, 0.3);
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.3);
     padding: 0 0 0 1ex;
     vertical-align: top;
 }
 
 #wc-user-menu {
     display: inline-block;
-    width: 30ex;
```

Comment:
Was there a particular reason for the width of the grey bar?

Reply:
I don't remember, I think this has something to do with a previous version of the default theme where this bar was used for more things. Probably the grey bar has to be removed, but your version is a step forward 👍.